velox/exec/tests/utils/Cursor.cpp

```diff
@@ -191,10 +191,37 @@ void TaskCursor::start() {
   }
 }
 
+namespace {
+bool waitForTaskDriversToFinish(
```

Comment:
Maybe we can make the existing waitForTaskDriversToFinish in velox/exec/tests/utils/QueryAssertions.h public.